e2e-tests/tests/headlamp.spec.ts

```diff
@@ -70,20 +70,46 @@ test('Service account tab should have headlamp-admin', async ({ page }) => {
   await securityPage.clickOnServiceAccountsSection();
 
   // Check if there is text "headlamp-admin" on the page
-  await headlampPage.checkPageContent('headlamp-admin');
+  await headlampPage.checkPageContent("headlamp-admin");
 });
 
-test('Logout the user', async ({ page }) => {
+test("Logout the user", async ({ page }) => {
   const headlampPage = new HeadlampPage(page);
 
   await headlampPage.authenticate();
   await headlampPage.logout();
 });
 
-test('404 page is present', async ({ page }) => {
+test("404 page is present", async ({ page }) => {
   const headlampPage = new HeadlampPage(page);
 
   await headlampPage.authenticate();
-  await headlampPage.navigateTopage('/404test', /Whoops! This page doesn't exist/);
+  await headlampPage.navigateTopage(
+    "/404test",
+    /Whoops! This page doesn't exist/
+  );
 });
+
+test("pagination goes to next page", async ({ page }) => {
+  const headlampPage = new HeadlampPage(page);
+  await headlampPage.authenticate();
+
+  await page.click('span:has-text("Security")');
+  await page.waitForLoadState("load");
+  await page.waitForSelector('span:has-text("Service Accounts")');
```

Comment:
We could use existing method on class 
```suggestion

  const securityPage = new SecurityPage(page);
  await securityPage.navigateToSecurity();
  await securityPage.clickOnServiceAccountsSection();
```